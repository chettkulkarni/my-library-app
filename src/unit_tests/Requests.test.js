import React from "react";
import { render } from "@testing-library/react";
import Requests from "../components/Requests";
import { BrowserRouter } from "react-router-dom";

jest.mock("firebase", () => ({
  auth: () => {
    return { onAuthStateChanged: val => true };
  }
}));
test("Snapshot for Requests", () => {
  const { container } = render(
    <BrowserRouter>
      <Requests />
    </BrowserRouter>
  );
  expect(container).toMatchSnapshot();
  console.log(6);
});
