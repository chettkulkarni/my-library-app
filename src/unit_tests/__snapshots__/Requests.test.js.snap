// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot for Requests 1`] = `
<div>
  <div
    class="container"
  >
    <div
      class="loading"
    />
  </div>
</div>
`;
